cloud/storage/core/libs/common/compressed_bitmap.cpp

```diff
@@ -83,21 +84,21 @@ class TCompressedBitmap::TImpl
             auto rangeInfo = NPrivate::RangeInfo<ui16, 64>(b, e);
 
             if (rangeInfo.First == rangeInfo.Last) {
-                count -= PopCount(Data[rangeInfo.First]);
+                count -= std::popcount(static_cast<ui64>(Data[rangeInfo.First]));
                 Data[rangeInfo.First] |= (MASK >> (64 - e + b)) << (b % 64);
-                count += PopCount(Data[rangeInfo.First]);
+                count += std::popcount(static_cast<ui64>(Data[rangeInfo.First]));
```

Comment:
why do we need `static_cast<ui64>` here and below isn't `Data[i]` already has ui64 type ?

Reply:
fixed